analytic_distribution_widget_remove_save/static/src/xml/analytic_distribution.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!-- Copyright 2023 ACSONE SA/NV
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<t
+    id="analytic_distribution_inherit"
+    t-inherit="analytic.AnalyticDistributionPopup"
+    t-inherit-mode="extension"
+    owl="1"
+>
+       <xpath expr="//div[@class='popupButtons']//span" position="replace">
+           <!-- Removing button -->
+       </xpath>
```

Comment:
```suggestion
       <!-- Removing button -->
       <xpath expr="//div[@class='popupButtons']//span" position="replace"/>
```

have you tried this?

Reply:
It also work, I have changed it. Thanks!